specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/applicationGateway.json

```diff
@@ -1843,6 +1850,23 @@
       },
       "description": "Header configuration of the Actions set in Application Gateway."
     },
+    "ApplicationGatewayUrlConfiguration": {
+      "properties": {
+        "modifiedPath": {
+          "type": "string",
+          "description": "It is the url path which user has provided for url rewrite. Null means no path will be updated. Default value is null."
```

Comment:
"It is" is kind of redundand, most descriptions in spec start would start with "The URL path..."